optimism/src/keccak/tests.rs

```diff
@@ -470,3 +474,52 @@ fn test_keccak_multiplicities() {
         2
     );
 }
+
+// Prover/Verifier test includidng the Keccak constraints
+#[test]
+fn test_keccak_prover() {
```

Comment:
This is the test that hashes a long enough preimage so that all the keccak steps are created and all the constraints can be evaluated (both on the witness side when computation happens, but also using the generic prover). For now, this is not checking the lookups yet, that will go in a separate PR.

Reply:
So I'm comparing this to https://keccak.team/keccak_specs_summary.html...

Is the "3 blocks" below meaning that we will do exactly 3 absorbs? 

Is `RATE_IN_BYTES` is the total size of `S` in that article? Or is that `CAPACITY_IN_BYTES`?..

Reply:
Yes, 3 blocks means it will need 3 absorbs. `RATE_IN_BYTES` is 136, so basically 1088 bits.